NewsApp/NewsApp/Controller/SearchController.swift

```diff
@@ -65,23 +70,26 @@ class SearchController: UITableViewController {
             guard let searchDateFromController = segue.destination as? SearchDateFromController else {
                 return
             }
+            searchDateFromController.dateFromSelected = self.parametersRequest.from
             searchDateFromController.delegate = self
         } else if segue.identifier == R.segue.searchController.segueToSearchDateTo.identifier {
             guard let searchDateToController = segue.destination as? SearchDateToController else {
                 return
             }
+            searchDateToController.dateToSelected = self.parametersRequest.to
             searchDateToController.delegate = self
         } else if segue.identifier == R.segue.searchController.segueToSearchLanguage.identifier {
             guard let searchLanguageController = segue.destination as? SearchLanguageController else {
                 return
             }
+            searchLanguageController.languageSelected = self.parametersRequest.language
```

Comment:
Est-ce qu'il y a pas moyen de regrouper les tests des else if et des guard let ?

Reply:
fix in "fix prepare"